src/Configurations/AccessRules.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Cloudflare\API\Configurations;
+
+class AccessRules implements Configurations
+{
+    private $configs = [];
+
+    public function setIP(string $value)
+    {
+        $this->configs[] = (object)['target' => 'ip', 'value' => $value];
```

Comment:
I could switch this to the following construct:
```php
$object = new \stdClass();
$object->target = "ip";
$object->value = $value;
```

But in my opinion this looks much nicer and shorter. I also used `$this->configs[] =` instead of `array_push` here, because it's faster and cleaner.